plugins/modules/network_compliance_workflow_manager.py

```diff
@@ -1098,27 +1133,50 @@ def get_task_status(self, task_id, task_name):
             If an error occurs during the API call, it will be caught and logged.
         """
         # Make an API call to retrieve the task tree
```

Comment:
This get_task_status also used in many modules.. Can we write in dnac.py and make use of it?

Reply:
Modified